packages/eventual-send/src/async-contexts/7-fluid-passing-style.js

```diff
@@ -0,0 +1,34 @@
+// @ts-nocheck
+/* eslint-disable no-underscore-dangle */
+// Given that the post-fps language has its own hidden
+// WeakMap, harden, undefined, assert
+
+// eslint-disable-next-line no-unused-vars
+export const _makeAsyncContext = F1 => {
+  const transposedMap = new WeakMap();
+
+  return harden({
+    run: (F2, _val, _cb, _args = []) => {
+      const key = harden({});
+      transposedMap.set(key, _val);
+      const F3 = harden(m => (m.has(key) ? m.get(key) : F2(m)));
```

Comment:
```suggestion
      const F3 = harden((F7, _m) => (_m.has(key) ? _m.get(key) : F2(_m)));
```

Reply:
Hi @jridgewell I don't think this is correct. Remember that this code is written manually in the post-fps language. The functions, keys, and maps involved at this line have no reality in the pre-fps language.